sample.env

```diff
@@ -8,5 +8,5 @@ REACT_APP_UNCHAINED_ETHEREUM_HTTP_URL=https://dev-api.ethereum.shapeshift.com
 REACT_APP_UNCHAINED_ETHEREUM_WS_URL=wss://dev-api.ethereum.shapeshift.com
 REACT_APP_UNCHAINED_BITCOIN_HTTP_URL=https://dev-api.bitcoin.shapeshift.com
 REACT_APP_UNCHAINED_BITCOIN_WS_URL=wss://dev-api.bitcoin.shapeshift.com
-REACT_APP_PORTIS_DAPP_ID=<get dapp ID from someone>
-REACT_APP_ETHEREUM_NODE_URL=<get free key from infura or other provider>
+REACT_APP_PORTIS_DAPP_ID=8609e6a8-e0dc-45e6-a0ad-edde63a4cdda
+REACT_APP_ETHEREUM_NODE_URL=https://eth-mainnet.alchemyapi.io/v2/lG_Rc1WYrmqrOUzTigl1CcSqlVAesAJl
```

Comment:
portis matches prod, we use an infura url starting with `d7` in prod - do we wanna use that? anyone can `ctrl-f` it in the prod build files if they want.

Reply:
done